src/runner/PlutoRunner.jl

```diff
@@ -253,6 +253,23 @@ module CantReturnInPluto
     replace_returns_with_error_in_interpolation(ex) = ex
 end
 
+# This is taken basically from fix_linenumbernodes! in Parse.jl
+function append_macrocellid_linenumbernodes!(ex::Expr, cell_id)
```

Comment:
This could be a method to `replace_pluto_properties_in_expr`

Reply:
Nice suggestion!